fury/molecular.py

```diff
@@ -0,0 +1,729 @@
+import vtk
+from vtk.util import numpy_support
+
+
+class Molecule(vtk.vtkMolecule):
+    """Your molecule class.
+
+    An object that is used to create molecules and store molecular data (e.g.
+    coordinate and bonding data).
+    This is a more pythonic version of ``vtkMolecule``.
+    """
+
+
+def add_atom(molecule, atomic_num, x_coord, y_coord, z_coord):
+    """Add atomic data to our molecule.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to which the atom is to be added.
+    atomic_num : int
+        Atomic number of the atom.
+    x_coord : float
+        x-coordinate of the atom.
+    y_coord : float
+        y-coordinate of the atom.
+    z_coord : float
+        z-coordinate of the atom.
+    """
+    molecule.AppendAtom(atomic_num, x_coord, y_coord, z_coord)
+
+
+def add_bond(molecule, atom1_index, atom2_index, bond_type=1):
+    """Add bonding data to our molecule. Establish a bond of type bond_type
+    between the atom at atom1_index and the atom at atom2_index.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to which the bond is to be added.
+    atom1_index : int
+        Index of the first atom.
+    atom2_index : int
+        Index of the second atom.
+    bond_type : int (optional)
+        Type of bond (single/double/triple). Default: 1
+    """
+    molecule.AppendBond(atom1_index, atom2_index, bond_type)
+
+
+def get_total_num_atoms(molecule):
```

Comment:
as said before, this should be a property.   In general, if you need a get/set, please, use property tag to be more pythonic